test/integration/golden_response/search_statvar/female.json

```diff
@@ -1,5 +1,33 @@
 {
   "statVars": [
+    {
```

Comment:
is this the order with which we will show stat var search results? if that's the case, should the new WHO vars be ranked lower than Count_Person_Female (which should be ranked above Count_Death_Female)?